.new_docs/en/build_and_install/cpu.md

```diff
@@ -33,12 +34,12 @@ make install
 
 ### Windows
 
-Windows编译需要满足条件
+For Windows, it needs: 
 
 - Windows 10/11 x64
 - Visual Studio 2019
 
-在Windows菜单中，找到`x64 Native Tools Command Prompt for VS 2019`打开，执行如下命令
+Open the `x64 Native Tools Command Prompt for VS 2019` in the windows menu and run the following commands: 
```

Comment:
Launch the `x64 Native Tools Command Prompt for VS 2019` from the Windows Start Menu and run the following commands: